data/json/professions.json

```diff
@@ -6265,5 +6305,890 @@
+      { "level": 4, "name": "gun" },
+      { "level": 4, "name": "smg" },
+      { "level": 3, "name": "pistol" },
+      { "level": 2, "name": "firstaid" },
+      { "level": 2, "name": "swimming" }
+    ],
+    "proficiencies": [ "prof_gunsmithing_improv", "prof_gunsmithing_basic", "prof_handloading", "prof_gun_cleaning" ],
+    "items": {
+      "both": {
+        "items": [ "tank_top", "hoodie", "pants_cargo", "socks", "sneakers", "press_dowel", "recipe_bullets" ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "smg_9mm", "ammo-item": "reloaded_9mmfmj", "charges": 20, "contents-item": "shoulder_strap" },
+          { "item": "survivor9mm_mag", "ammo-item": "reloaded_9mmfmj", "charges": 20 },
+          { "item": "survivor9mm_mag", "ammo-item": "reloaded_9mmfmj", "charges": 20 },
+          { "item": "reloaded_9mmfmj", "charges": 20 },
+          { "item": "shot_scrap", "charges": 10 },
+          { "item": "ashot", "ammo-item": "shot_scrap", "charges": 1, "container-item": "XL_holster" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bbgunner",
+    "name": "BB Gunner",
+    "description": "You were the bane of the neighbourhood, causing all sorts of mischief with the BB gun you got for your birthday.  Your streak of prankishness has come to an end, however, as there are no adults left to pester.",
+    "points": 0,
+    "skills": [ { "level": 2, "name": "gun" }, { "level": 1, "name": "rifle" } ],
+    "items": {
+      "both": {
+        "items": [ "hoodie", "jeans", "socks", "sneakers", "wristwatch" ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "bbgun", "ammo-item": "bb", "charges": 150, "contents-item": "shoulder_strap" },
+          { "item": "tshirt", "variant": "generic_tshirt" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "boy_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "exotic_dancer",
+    "name": "Exotic Dancer",
+    "description": "As you were giving a private dance to a lecherous patron, he tried to take a bite out of you.  You had to leave your everyday clothes behind as you made your way straight to the exit.  Now you're all alone with barely anything to your name.",
+    "points": 0,
+    "skills": [ { "level": 4, "name": "dodge" } ],
+    "flags": [ "NO_BONUS_ITEMS" ],
+    "traits": [ "DEFT" ],
+    "items": { "male": [ "bikini_bottom_short" ], "female": [ "bikini_top_short", "bikini_bottom_short", "heels" ] }
+  },
+  {
+    "type": "profession",
+    "id": "bank_robber",
+    "name": "Trigger Man",
+    "description": "The bank robbery was going exactly according to plan… until the brains of the operation got bit by the inside man.  To add insult to injury, your getaway driver decided to bail on you.  At least you don't have to split the haul now.",
```

Comment:
Where did the haul go in this prof's inventory 🤔

Reply:
{ "item": "duffelbag", "contents-group": "bank_score" },

 {
    "type": "item_group",
    "subtype": "collection",
    "id": "bank_score",
    "entries": [
      { "item": "money_strap_hundred", "count": 20 },
      { "item": "money_strap_fifty", "count": 20 },
      { "item": "money_strap_twenty", "count": 20 }
    ]
  },

Reply:
I can't read :D